src/vuestic-theme/vuestic-components/vuestic-navbar/VuesticNavbar.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <nav class="navbar app-navbar navbar-toggleable-md">
+    <div class="navbar-logo-container">
+      <router-link class="navbar-brand" to="/dashboard">
+        <slot name="logo"></slot>
+      </router-link>
+    </div>
+    <div class="row navbar-container">
+      <div class="header-elem-wrapper">
+        <slot name="selector"></slot>
+      </div>
+      <div class="header-elem-wrapper navbar-text offset-md-1 col-md-7">
+        <slot name="center"></slot>
+      </div>
+      <slot></slot>
+    </div>
+  </nav>
+</template>
+
+<script>
+  export default {
+    name: 'vuestic-navbar',
+  }
+</script>
+
+<style lang="scss">
+  @import "../../../sass/_variables.scss";
+  @import "~bootstrap/scss/mixins/breakpoints";
+  @import "~bootstrap/scss/functions";
+  @import "~bootstrap/scss/variables";
+
+  .navbar {
+    .layout-fixed & {
+      position: fixed;
+      top: 0;
+      width: 100%;
+      z-index: 1;
+    }
+    .header-elem-wrapper {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+  }
+
+  .navbar.app-navbar {
```

Comment:
Dependency goes other way round. `app-navbar` is parent. Shouldn't be mentioned in child.